src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/DividerSamplePage.xaml

```diff
@@ -0,0 +1,131 @@
+﻿<Page x:Class="Uno.Material.Samples.Content.Controls.DividerSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:controls="using:Uno.Material.Controls"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  mc:Ignorable="d">
+
+	<Page.Resources>
+		<DataTemplate x:Key="DefaultItemTemplate">
+			<Grid Width="200"
+				  Padding="8">
+				<Grid.ColumnDefinitions>
+					<ColumnDefinition Width="*" />
+					<ColumnDefinition Width="Auto" />
+				</Grid.ColumnDefinitions>
+
+				<TextBlock Text="Item"
+						   Foreground="{StaticResource MaterialOnBackgroundBrush}" />
+
+				<SymbolIcon Symbol="Favorite"
+							Foreground="{StaticResource MaterialOnBackgroundBrush}"
+							Grid.Column="1" />
+			</Grid>
+		</DataTemplate>
+
+		<DataTemplate x:Key="CustomItemTemplate">
+			<Grid Width="200"
+				  Padding="8">
+				<Grid.ColumnDefinitions>
+					<ColumnDefinition Width="*" />
+					<ColumnDefinition Width="Auto" />
+				</Grid.ColumnDefinitions>
+
+				<TextBlock Text="Item"
+						   Foreground="{StaticResource MaterialOnPrimaryBrush}" />
+
+				<SymbolIcon Symbol="Favorite"
+							Foreground="{StaticResource MaterialOnPrimaryBrush}"
+							Grid.Column="1" />
+			</Grid>
+		</DataTemplate>
+	</Page.Resources>
+
+	<ScrollViewer Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<StackPanel>
+
+			<smtx:XamlDisplay UniqueKey="Divider_Default">
+				<StackPanel Background="{StaticResource MaterialSurfaceBrush}"
+							Margin="8">
+					<ContentControl ContentTemplate="{StaticResource DefaultItemTemplate}" />
+
+					<controls:Divider Style="{StaticResource MaterialDividerStyle}" />
+
+					<ContentControl ContentTemplate="{StaticResource DefaultItemTemplate}" />
+				</StackPanel>
+			</smtx:XamlDisplay>
+
+
+			<smtx:XamlDisplay UniqueKey="Divider_DefaultWithSubHeader">
+				<StackPanel Background="{StaticResource MaterialSurfaceBrush}"
+							Margin="8">
+
+					<ContentControl ContentTemplate="{StaticResource DefaultItemTemplate}" />
+
+					<controls:Divider Style="{StaticResource MaterialDividerStyle}"
+									  SubHeader="SubHeader" />
+
+					<ContentControl ContentTemplate="{StaticResource DefaultItemTemplate}" />
+				</StackPanel>
+			</smtx:XamlDisplay>
+
+			<smtx:XamlDisplay UniqueKey="Divider_Custom">
+				<StackPanel Background="{StaticResource MaterialPrimaryBrush}"
+							Margin="8">
+
+					<ContentControl ContentTemplate="{StaticResource CustomItemTemplate}" />
+
+					<controls:Divider Style="{StaticResource MaterialDividerStyle}"
+									  Foreground="LightGray" />
```

Comment:
foreground should be OnPrimaryBrush?

Reply:
or OnPrimaryMediumBrush

Reply:
Shouldn't be OnSurfaceBrush?

Reply:
OnSurfaceBrush is already the default color. I though it would be a good idea to display that we can have custom colors for the divider control. In this case, I was trying to go with the blue color to make it look like the owl sample on the divider documentation https://material.io/components/dividers#theming

Reply:
I forgot to mention, I choose OnPrimaryLowBrush, because I thought it was more appropriate for the border